src/Security/CMSSecurity.php

```diff
@@ -142,19 +162,6 @@ protected function preLogin()
         return parent::preLogin();
     }
 
-    public function GetLoginForms()
```

Comment:
Unused method in the whole codebase of SilverStripe now. Removed for that reason.